prow/plugins/branch-updater/branch-updater.go

```diff
@@ -0,0 +1,164 @@
+package branch_updater
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const (
+	gitHubContextNameTide        = "tide"
+	gitHubMergeableStateBehind   = "behind"
+	gitHubMergeStateRefreshDelay = 10
+	pluginName                   = "branch-updater"
+	tideContextStatusSuccess     = "success"
+)
+
+var (
+	// This plugin should ignore any event type not on this list.
+	enabledEvents = []github.PullRequestEventAction{
+		github.PullRequestActionOpened,
+		github.PullRequestActionEdited,
+		github.PullRequestActionReadyForReview,
+		github.PullRequestActionReopened,
+	}
+)
+
+type githubClient interface {
+	UpdatePullRequestBranch(owner, repo string, number int, expectedHeadSha *string) error
+	GetPullRequest(owner, repo string, number int) (*github.PullRequest, error)
+	GetCombinedStatus(org, repo, ref string) (*github.CombinedStatus, error)
+}
+
+func helpProvider(config *plugins.Configuration, enabledRepos []config.OrgRepo) (*pluginhelp.PluginHelp, error) {
+	configInfo := map[string]string{}
```

Comment:
TL;DR in this case, these [are, in fact](https://stackoverflow.com/questions/53575693/difference-between-makemap-and-map), equivalent.

The difference is that the `make()` syntax allows you to pre-define a specific length of the map if you know it ahead of time, so that the map won't have to be resized as items are added later on. It's a performance optimization for maps that are larger, have more dynamic sizes, or are particularly performance-critical.

None of those are true here, so it doesn't particularly matter. I took this particular syntax from elsewhere in the Prow codebase for consistency, but this is sort of a style-guide thing more than anything.